charts/bctlquickstart/values.yaml

```diff
@@ -10,14 +10,12 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: "2.0.0" # For our quickstart job
   agentTag: "latest" # For our actual agent
```

Comment:
nit: can we rename this too `agentImageTag` to be consistent. And also update `tag` to be `quickstartTag`

Reply:
agreed lets rename these helm values for clarity